opennlp-tools/src/main/java/opennlp/tools/formats/brat/BratDocumentParser.java

```diff
@@ -145,5 +148,13 @@ public BratDocumentParser(SentenceDetector sentenceDetector, Tokenizer tokenizer
 
     return samples;
   }
+
+  private boolean isSpanAnnotation(BratAnnotation ann) {
+    if (ann instanceof SpanAnnotation &&
```

Comment:
Probably better to be clear about what we expect:
- Don't enable name type filtering, in that case pass null as nameTypes parameter
- If an empty set is passed an IllegalArgumentException could be thrown, doesn't make sense to filter all